runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerItems.java

```diff
@@ -388,6 +388,7 @@
 	ZAMORAK_GODSWORD(ItemID.ZAMORAK_GODSWORD, 8),
 	ZAMORAK_GODSWORD_OR(ItemID.ZAMORAK_GODSWORD_OR, 8),
 	IVANDIS_FLAIL(ItemID.IVANDIS_FLAIL, 5),
+    ANCIENT_STAFF(ItemID.ANCIENT_STAFF, -1),

```

Comment:
Correct indentation to a tab and move to the top of the list, as they are sorted by prayer bonus.

Reply:
Okay. I need to stop long enough to setup my defaults properly. Then I'll go reorder the list.

Reply:
Alright, the indentation issues should be done with now.